experimental/Schemes/Resolution_structure.jl

```diff
@@ -1,61 +1,209 @@
-## Warnung: show auf desingMor geht noch nicht!!!
 export _desing_curve
```

Comment:
```suggestion
```
Internal functions (as indicated by the leading underscore) should not get exported

Reply:
done